docs/protocol/specifications/vm1.md

```diff
@@ -75,8 +75,8 @@ Execution of any of the following instructions results in immediate failure:
 * [AMOUNT](#amount)
 * [MINTIME](#mintime)
 * [MAXTIME](#maxtime)
-* [TXREFDATAHASH](#txrefdatahash)
-* [REFDATAHASH](#refdatahash)
+* [TXDATAHASH](#txdatahash)
```

Comment:
As has been discussed elsewhere, whether or not this is a hash is an app-level decision. We should leave hash out of the name and just call this `TXDATA`. (`DATAHASH` can be `INPUTDATA` if just plain `DATA` is too generic.)